lib/stream-chat/types.rb

```diff
@@ -0,0 +1,13 @@
+# typed: strict
+# frozen_string_literal: true
+
+module StreamChat
+  extend T::Sig
+  T::Configuration.default_checked_level = :never
+  # For now we disable runtime type checks.
+  # We will enable it with a major bump in the future,
+  # but for now, let's just run a static type check.
+
+  StringKeyHash = T.type_alias { T::Hash[T.any(String, Symbol), T.untyped] }
```

Comment:
We had issues with having strings/symbols in maps before, would you think we should allow only symbols here in next major version?

Reply:
Agreed. Want me to modify this to this?

```
SymbolKeyHash = T.type_alias { T::Hash[Symbol, T.untyped] }
```

Reply:
Yes, I think it would be right move.